sdk/core/Azure.Core.Experimental/src/PopTokenAuthenticationPolicy.cs

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Azure.Core.Pipeline
+{
+    /// <summary>
+    /// A policy that supports Proof of Possession (PoP) tokens.
+    /// </summary>
+    public class PopTokenAuthenticationPolicy : HttpPipelinePolicy
+    {
+        private readonly ISupportsProofOfPossession _tokenCredential;
+        private string? _popNonce;
+        private readonly string[] _scopes;
+
+        /// <summary>
+        /// Creates a new instance of <see cref="PopTokenAuthenticationPolicy"/>.
+        /// </summary>
+        /// <param name="credential"></param>
```

Comment:
Include a description for this param. That way, something meaningful displays in the ctor's supporting API ref page on learn.microsoft.com.